src/erlfmt_parse.yrl

```diff
@@ -1014,7 +1016,15 @@ parse_form([{'-',A1},{atom,A2,define}|Tokens]) ->
             parse(NewTokens3)
     end;
 parse_form(Tokens) ->
-    parse(Tokens).
+    case parse(Tokens) of
+        {ok, _} = Res ->
+            Res;
+        Error ->
+            case parse([{standalone_expr, #{}} | Tokens]) of
```

Comment:
I would create a function parse_expr that does this and then a function parse_any that tries parse_form and then parse_expr, since the name `parse` is already taken.  Now I find it slightly misleading, since parse_form implies that it will only parse a form, but maybe I am missing something?